Products/CMFPlone/controlpanel/browser/controlpanel_usergroups_layout.pt

```diff
@@ -8,16 +8,16 @@
 
 <body>
 
+<div metal:use-macro="context/global_statusmessage/macros/portal_message">
+      Portal status message
+    </div>
```

Comment:
@rohnsha0 This won't do anything. The template is using a macro (see line 6) and filling a slot in that macro (see line 15). Anything in this file that is not inside the slot will not be used.

But, after looking for a couple minutes I don't yet understand why this template needs to customize the normal display of the status message at all, so this might need more investigation.

Reply:
As the alert message is placed above the nav bar.... 
But see in the screenshots for the issue for reference!
I think that's misplaced!
Correct me if I'm wrong!